Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -110,6 +141,26 @@ void ABenchmarkGymGameMode::Tick(float DeltaSeconds)
 				}
 			}
 		}
+
+		for (int i = AIControlledPlayers.Num() - 1; i >= 0; i--)
+		{
+			if (AIControlledPlayers[i] == nullptr)
+			{
+				AIControlledPlayers.RemoveAt(i);
+				AIControllerPlayerRunIndexes.RemoveAt(i);
+				continue;
+			}
+
+			ACharacter* Player = AIControlledPlayers[i]->GetCharacter();
+			int Info = AIControllerPlayerRunIndexes[i];
+			if (UDeterministicBlackboardValues* Blackboard = (UDeterministicBlackboardValues*)Player->FindComponentByClass(UDeterministicBlackboardValues::StaticClass()))
+			{
+				auto& Points = PlayerRunPoints[Info % PlayerRunPoints.Num()];
```

Comment:
Why the %? Shouldn't there be as many runpoints as there are players?

Reply:
There should be, although there isn't anything to prevent spawning too many players either on a local build or in the cloud. Perhaps there should be some warning log here as well